site/docs/try/releases.md

```diff
@@ -1,5 +1,13 @@
 # Releases
 
+## 0.21.0 Release (March 1, 2022)
+
+* Add tracing to database-adapter internals
+* Introduce compatibility and upgrade tests
+* Refactor StreamingUtil class
+* Support for Spark 3.1 + 3.2.1 in Nessie SQL extensions
+* Proper usage of commit-id in Spark SQL extensions
```

Comment:
should we mention `Add DELETE_DEFAULT_BRANCH access check`?

Reply:
done